cross_compile/Dockerfile_workspace

```diff
@@ -97,7 +97,7 @@ RUN rosdep install --from-paths src /opt/ros/${ROS_DISTRO}/share \
 # Ignore some package that have known errors
 RUN touch \
     src/ros2/rviz/COLCON_IGNORE \
-    src/ros-visualization/COLCON_IGNORE
+    src/ros-visualization/COLCON_IGNORE; exit 0
```

Comment:
Why do we need this?

Reply:
Theses packages don't compile on armhf. 
If they exist, we don't want to compile them.
We add `exit 0` because if the `touch` command files (i.e. packages don't exist) then we don't want the whole build process to fail.